hl-alyx-sdk-setup.bat

```diff
@@ -1,105 +1,97 @@
 
 @ECHO OFF
 ECHO NOTE
+ECHO Make sure you have enough space, about double the game size!
 ECHO Please follow instructions at the end of script to extract / download additional needed files!
 ECHO :::::::::::::::::::::::::::::::::::::::
-SET steam_default_dir=C:/Program Files (x86)/Steam/steamapps
+pause
 :: Steam directory check
-SET /P steam_defaults=Use default steam directory %steam_default_dir% (Y\N)?
-IF /I NOT "%steam_defaults%"=="Y" GOTO SetupSteamDir
-GOTO:file_copy_prompt
+:: Getting Steam path from registry
+for /f "usebackq tokens=1,2,*" %%i in (`reg query "HKCU\Software\Valve\Steam" /v "SteamPath"`) do set "steampath=%%~k"
+:: Replacing "/"'s with "\" in some cases
+set steampath=%steampath:/=\%
+:: Testing common paths
+if not exist %steampath%\steam.exe (
+	if not exist "%ProgramFiles(x86)%\steam\\steam.exe" (
+		if not exist "%ProgramFiles%\steam\steam.exe" (
+			goto DontRun
+		) else (
+			set steampath=%ProgramFiles%\steam
+		)
+	) else set steampath=%ProgramFiles(x86)%\steam )
```

Comment:
There should be a newline after the steam here, currently it's setting the steam path as .../steam )